app/models/user.rb

```diff
@@ -11,6 +13,11 @@ def full_name
     first_name + " " + last_name
   end
 
+  def highest_role
+    return "a #{roles.last.name}" if !roles.empty?
```

Comment:
A possible refactor.

if roles.empty?
 "an Administrator"
else
 "a #{roles.last.name}"
end

Reply:
This will change further down the line when we begin to incorporate other role types. For right now there's only 'Platform Admin'. If a user.roles is empty, then that's indicative that they're not an admin of any sort. If user.roles is _not_  empty, then they're an admin as the only role that exists is Platform Admin. This will become more complex and specific once we begin to add more roles.